framework/src/controller/helpers/configurator/configurator.js

```diff
@@ -0,0 +1,148 @@
+const _ = require('lodash');
```

Comment:
can we move this folder to
`controller/configurator/configurator.js`

Reply:
I am convinced its a utility not a main namespace. As we planned to rename `helpers` to `util` then it will look more appropriate.

Reply:
utility but it's one of the main functionality for controller, and it has specific logic init, so i think it's worth to put it as first citizen.
Also, i want to get rid of `helpers` or `utils` in general =)

Reply:
To me its not a main functionality of the controller, rather add on to simplify configuration management for end user.  These functions still exists and must required, call these helpers or util. In any large project you always have such collection of functions. And `util` is a known convention in NodeJS community so not a bad thing to have. 

And wether we treat these as first citizen or not that depends how much we test coverage we have for them.